azure-devops/validate-files.cmd

```diff
@@ -0,0 +1,4 @@
+:: Copyright (c) Microsoft Corporation.
+:: SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"%1"
+@echo If your build fails here, you need to fix the listed issues.

```

Comment:
I don't think this .cmd adds value since you already print the right bits from your .exe. I only needed the .cmd to adapt to clang-format.exe (and then because I made `parallelize.exe` a generic thing, not a specific thing)

Reply:
I will attempt to invoke the exe directly.

Reply:
I had to revert that change because it totally damages the "Validation failure" output - possibly because it isn't buffered, and something is inserting a bunch of newlines somehow.